src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordCommon.cs

```diff
@@ -154,7 +142,6 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                     ref positionalParamSymbols,
                     constructorSymbols[primaryIndex].Parameters))
             {
-                // grab parameter defaults and reorder positional param info to be in order of primary constructor params
```

Comment:
Why the comment removal?

Reply:
Not sure, it must have been missed during some merge conflict resolution or moving code over to another file. Put it back in